lib/ReactViews/Mobile/MobileHeader.jsx

```diff
@@ -194,11 +193,16 @@ const MobileHeader = observer(
                   </RawButton>
                   <Branding
                     terria={this.props.terria}
+                    viewState={this.props.viewState}
                     version={this.props.version}
-                    displayOne={displayOne}
                   />
                 </Box>
-                <div className={Styles.groupRight}>
+                <div
+                  className={Styles.groupRight}
+                  css={`
+                    background-color: ${p => p.theme.dark};
+                  `}
+                >
```

Comment:
This adds background behind Mobile header to cover logo - like so:

![image](https://user-images.githubusercontent.com/6187649/115509976-80037f00-a2c2-11eb-820c-43e0cf811941.png)

vs

![image](https://user-images.githubusercontent.com/6187649/115510002-872a8d00-a2c2-11eb-966c-9a0839d5ad49.png)